dumpgenerator.py

```diff
@@ -1058,53 +1065,85 @@ def generateImageDump(config={}, other={}, images=[], start='', session=None):
     if not os.path.isdir(imagepath):
         print 'Creating "%s" directory' % (imagepath)
         os.makedirs(imagepath)
-
+    
+    maxseconds = 100  # max seconds to wait in a single sleeping
```

Comment:
It's currently a fixed value for the XML part as well.
I've only followed that. The number of retries is configurable though.